dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -224,6 +224,20 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 		"integration/docker/Dockerfile.fuse",
 		"integration/docker/entrypoint.sh",
 		"integration/fuse/bin/alluxio-fuse",
+		fmt.Sprintf("lib/alluxio-underfs-adl-%v.jar", version),
```

Comment:
Forget those lines, are those needed to be added?

Reply:
If not, when resolving the conflicts, please remove those lines

Reply:
> If not, when resolving the conflicts, please remove those lines

Sorry, this lines are needed to be removed!